src/app/components/InRadius/InRadius.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+type InRadiusProps = {
+  getRadius: (radius: number) => void
+}
+
+export default function InRadius({ getRadius }: InRadiusProps): JSX.Element {
+  const [radius, setRadius] = useState(0)
+
+  getRadius(radius)
+
+  return (
+    <>
+      <RadiusText>
+        Search Radius{' '}
+        <RadiusInput
+          type="number"
```

Comment:
You could use [range](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) here :)

Reply:
great, thats what i needed, and its easyer than I thought!